apps/shared/avifpng.c

```diff
@@ -312,6 +307,86 @@ avifBool avifPNGRead(const char * inputFilename,
         *outPNGDepth = imgBitDepth;
     }
 
+    if (!ignoreProfile) {
+        char * iccpProfileName = NULL;
+        int iccpCompression = 0;
+        unsigned char * iccpData = NULL;
+        png_uint_32 iccpDataLen = 0;
+        int srgbIntent = 0;
+
+        // PNG specification 4.2.2: The sRGB and iCCP chunks should not both appear.
```

Comment:
Nit: Add "Section" before "4.2.2", otherwise "4.2.2" may be misinterpreted as a version number.